reinforcement_learning/actor_critic.py

```diff
@@ -56,6 +56,7 @@ def forward(self, x):
 def select_action(state):
     state = torch.from_numpy(state).float().unsqueeze(0)
     probs, state_value = model(Variable(state))
+    state_value = state_value.squeeze(0)
```

Comment:
I think this will work if you just get rid of the unsqueeze above.

Reply:
Done